contentcuration/contentcuration/frontend/channelEdit/components/sidePanels/SubmitToCommunityLibrarySidePanel/index.vue

```diff
@@ -130,6 +130,36 @@
                 {{ detectedCategories }}
               </LoadingText>
             </div>
+            <div
+              v-if="licenseAuditIsLoading && !isPublishing"
+              class="license-audit-loader"
+            >
+              <KCircularLoader disableDefaultTransition />
+              <div class="audit-text-wrapper">
+                <div class="audit-text-primary">
+                  {{ checkingChannelCompatibility$() }}
+                </div>
+                <div class="audit-text-secondary">
+                  {{ checkingChannelCompatibilitySecondary$() }}
+                </div>
+              </div>
+            </div>
+            <LicenseStatus
+              v-if="licenseAuditIsFinished && !isPublishing"
+              :invalid-licenses="invalidLicenses"
+              :included-licenses="includedLicenses"
+            />
+            <SpecialPermissionsList
+              v-if="
+                licenseAuditIsFinished &&
+                  specialPermissions &&
+                  specialPermissions.length > 0 &&
+                  !isPublishing
+              "
+              v-model="checkedSpecialPermissions"
+              :permission-ids="specialPermissions"
```

Comment:
Just noting that, although this is absolutely correct, we usually use the camel case for component props, so we would usually use `:permissionIds="specialPermissions"` instead.